content/applications/sales/sales/products_prices/products/product_images.rst

```diff
@@ -1,130 +1,168 @@
-===================================================
-Automatically get product images with Google Images
-===================================================
+=================================
+Product images with Google Images
+=================================
 
-The product images are very useful in Odoo, for example, to quickly find a product or check if you
-scanned the right one, but it can be a bit painful to set up especially if you have a lot of
-products. **Google Custom Search** allows finding images automatically for your product, based on
-their barcode, keeping your focus on what matters in your business.
+Having appropriate product images in Odoo is useful for a number of reasons. However, if a lot of
+products need images, it can become incredibly time-consuming.
+
+But, with *Google Custom Search* API configured within the Odoo database, finding product images for
+products (based on their barcode) is extremely efficient.
 
 .. _product_images/configuration:
 
 Configuration
 =============
 
-This functionnality requires configuration both on Google and on Odoo.
+In order to utilize *Google Custom Search* within an Odoo database, configuration **must** be
+completed on both -- Google and Odoo.
 
-With a free Google account, you can get up to 100 free images per day. If you need a higher rate,
-you'll have to upgrade to a billing account.
+.. note::
+   Free Google accounts allow up to 100 free images per day. If a higher amount is needed, a billing
```

Comment:
What are you allowed to do with the images? Select them? I think the language here could be a bit more specific.
```suggestion
   Free Google accounts allow up to 100 free images per day.
```